docs/release-v1-perf-scale-benchmarks.md

```diff
@@ -0,0 +1,71 @@
+### Benchmarks
+
+This document shares some of the performance benchmarks observed as part of the v1 release.
+It will talk about what this means for user's perf and scale story.
+
+#### Background: How to interpret the numbers?
+
+KubeVirt is an extension for Kubernetes that includes a collection of custom resource definitions
+(CRDs) served by kubevirt-apiserver. These CRDs are managed by controllers. Due to the distributed
+nature of the system, understanding performance and scalability data becomes challenging without
+taking specific assumptions into account. This section aims to provide clarity on those assumptions.
+
+1. The data presented in the document is collected by `periodic-kubevirt-e2e-k8s-1.25-sig-performance`.
+1. The test creates 100 minimal VMIs, with a small pause of 100 ms between creation of 2 VMIs. The definition
```

Comment:
That's one of three tests ran as part of this job, can you expand this to include the `[small] create a batch of %d running VMs` and `[small] create a batch of %d running VMs using a single instancetype and preference` tests as well?

Reply:
Apologies @lyarwood, the perf-report-creator, tool used to scrape the data, is lacking extension for instance types. I have created an issue for this https://github.com/kubevirt/project-infra/issues/2874. If it is okay with you, we can add it post v1 to track performance for next release

Reply:
@lyarwood I have added verbose details about 2/3 tests being covered and which metrics represent which test, PTAL